src/media/media.service.spec.ts

```diff
@@ -116,4 +119,31 @@ describe('media service', () => {
       );
     });
   });
+
+  describe('getMedia method', () => {
```

Comment:
These methods should be changed if you will follow my recommendations to write the logic of getting the filename from system in the service instead of the repository.

Reply:
https://github.com/m3ntorship/pickify-media/pull/12#discussion_r661320844